airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -114,58 +101,58 @@ const SideBar: React.FC = () => {
         <Menu>
           {workspace.displaySetupWizard ? (
             <li>
-              <MenuItem to={RoutePaths.Onboarding}>
+              <NavLink className={({ isActive }) => menuItemStyle(isActive)} to={RoutePaths.Onboarding}>
                 <OnboardingIcon />
                 <Text>
                   <FormattedMessage id="sidebar.onboarding" />
                 </Text>
-              </MenuItem>
+              </NavLink>
             </li>
           ) : null}
           <li>
-            <MenuItem to={RoutePaths.Connections}>
+            <NavLink className={({ isActive }) => menuItemStyle(isActive)} to={RoutePaths.Connections}>
```

Comment:
There is a lot of repetition of this logic. Maybe this can be either component or alternatively can be instanced a single class name that can be shared with all the components. For example:

```ts
const navLinkClassName = ({ isActive }) => menuItemStyle(isActive);
...
<NavLink className={navLinkClassName} to={RoutePaths.Connections}>
```